ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent-nonprod.json

```diff
@@ -124,5 +124,12 @@
     "CaseEventID": "populateCase-PREPARE_FOR_HEARING",
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "addFamilyManCaseNumber-superuser",
```

Comment:
maybe editFamilyManCaseNumber instead of addFamilyManCaseNumber-superuser

Reply:
sure makes more sense, can be more explicit with the end state in the submitted file as well